xmtp_id/src/associations/unsigned_actions.rs

```diff
@@ -14,58 +19,52 @@ pub struct UnsignedCreateInbox {
 
 impl SignatureTextCreator for UnsignedCreateInbox {
     fn signature_text(&self) -> String {
-        format!(
-            // TODO: Finalize text
-            "Create Inbox: {}",
-            generate_inbox_id(&self.account_address, &self.nonce)
-        )
+        format!("- Create inbox\n(Owner: {})", self.account_address)
     }
 }
 
 #[derive(Clone)]
 pub struct UnsignedAddAssociation {
-    pub inbox_id: String,
     pub new_member_identifier: MemberIdentifier,
 }
 
 impl SignatureTextCreator for UnsignedAddAssociation {
     fn signature_text(&self) -> String {
+        let (target_kind, id_kind) = get_terms_for_kind(self.new_member_identifier.kind());
         format!(
-            // TODO: Finalize text
-            "Add {} to Inbox {}",
-            self.new_member_identifier, self.inbox_id
+            "- Grant messaging access to {target_kind}\n({id_kind}: {})",
+            self.new_member_identifier
```

Comment:
I think the change made to `RevokeAssociation` needs to be applied here

Reply:
Missed that one